app/model/commands/PublishAtomCommand.scala

```diff
@@ -92,20 +89,68 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
     }
   }
 
-  private def updateYouTube(atom: MediaAtom, asset: Asset): Unit = {
+  private def updateYouTube(previewAtom: MediaAtom, asset: Asset): MediaAtom = {
+    updateYoutubeMetadata(previewAtom, asset)
+    updateYoutubeThumbnail(previewAtom, asset)
+    createOrUpdateYoutubeClaim(previewAtom, asset)
+  }
+
+  private def createOrUpdateYoutubeClaim(previewAtom: MediaAtom, asset: Asset): MediaAtom = {
+    if (previewAtom.blockAds.isEmpty) {
```

Comment:
`previewAtom.blockAds.isEmpty` is empty after we initial create an atom. Doing this here saves a database call as we know that it also hasn't been published.